g2p_seq2seq/data_utils.py

```diff
@@ -127,7 +127,8 @@ def load_params(model_path):
   """
   # Checking model's architecture for decode processes.
   if gfile.Exists(os.path.join(model_path, "model.params")):
-    params = open(os.path.join(model_path, "model.params")).readlines()
+    with open(os.path.join(model_path, "model.params")) as f:
```

Comment:
Shouldn't we put everything below inside with? Otherwise params will be undefined?